content/influxdb/cloud/tools/grafana.md

```diff
@@ -70,35 +70,74 @@ configure your InfluxDB connection:
 {{% tab-content %}}
 
 ## Configure Grafana to use InfluxQL
+To query InfluxDB Cloud from Grafana using InfluxQL:
 
-With **InfluxQL** selected as the query language in your InfluxDB data source,
-configure your InfluxDB connection:
+1. [Create an InfluxDB DBRP mapping](#create-an-influxdb-dbrp-mapping)
+2. [Configure your InfluxDB connection](#configure-your-influxdb-connection)
+
+### Create an InfluxDB DBRP mapping
+When using InfluxQL to query InfluxDB Cloud, the query must specify a database and a retention policy.
+Use the [`influx v1 dbrp create` command](/influxdb/cloud/reference/cli/influx/v1/dbrp/create/)
+command to create a database/retention policy (DBRP) mapping that maps a database
+and retention policy combination to an InfluxDB Cloud [bucket](/influxdb/cloud/reference/glossary/#bucket).
+
+{{% note %}}
+##### Automatically create DBRP mappings on write
+When using the InfluxDB 1.x compatibility API to write data to InfluxDB Cloud,
+InfluxDB Cloud automatically creates DBRP mappings for buckets whose names match the
+`db/rp` naming pattern of the database and retention policy specified in the write request.
+For more information, see [Database and retention policy mapping – Writing data](/influxdb/cloud/reference/api/influxdb-1x/dbrp/#when-writing-data).
+{{% /note %}}
+
+Provide the following:
+
+- database name
+- retention policy
+- [bucket ID](/influxdb/cloud/organizations/buckets/view-buckets/)
+
```

Comment:
The command requires an org and token as well.  That may be obvious to Influxers but I think it should be explicitly called out here.  That could be in the form of prefacing these instructions with a suggestion/recommendation that they set up an `influx config` first and use that....or actually including those flags as part of the command in these instructions.

Reply:
I added instructions for setting up CLI configs in the section about installing the CLI mentioned above.

Reply:
Sweet, seems like a good place for it!